lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -547,6 +566,22 @@ namespace Js
 #include "IntlExtensionObjectBuiltIns.h"
 #undef INTL_ENTRY
 
+        library->AddMember(intlNativeInterfaces, PropertyIds::FallbackSymbol, library->CreateSymbol(BuiltInPropertyRecords::_intlFallbackSymbol));
+
+        DynamicObject * enumObj = nullptr;
+
+// Projects the exact layout of our C++ enums into Intl.js so that we dont have to remember to keep them in sync
+#define ENUM_VALUE(enumName, unicodeKey, value) library->AddMember(enumObj, PropertyIds::##unicodeKey, JavascriptNumber::ToVar(value, scriptContext));
```

Comment:
`unicodeKey` should really be `propId` or something similar -- I will change it after I rebase.